src/frontend/components/SøknadsSteg/Forside/Forside.tsx

```diff
@@ -48,27 +52,83 @@ const Forside: React.FC = () => {
         mellomlagretVerdi && mellomlagretVerdi.modellVersjon === Miljø().modellVersjon;
 
     return (
-        <InnholdContainer>
-            <Box marginBlock="0 12">
-                <Heading size="xlarge" align="center">
-                    <TekstBlock block={soeknadstittel} />
+        <Layout>
```

Comment:
Skal denne eksplisitt være annerledes enn layoutet `InnholdContainer` som brukes på alle stegene ++? I så fall, hvorfor det?

Reply:
Både Layout og InnholdContainer setter bredden på innholdet og legger til marginer.

Forskjellen er at InnholdContainer i tillegg definerer flex relatert styling, mens på forsiden hvor Layout brukes, bruker vi heller VStack istedenfor manuelt oppsett av flex relatert styling som i InnholdContainer. Grunnen til dette er at InnholdContainer ikke definerer flex-gap, og heller bruker bunn-marginer på hver komponentgruppe for å skape space mellom komponenter, noe som fører til mye ekstra kode. I Layout håndteres spacing med gap i VStack.

Begge disse skal senere byttes ut med Aksel sin Page komponent ettersom dette vil definere side/page-strukturen bedre enn Layout og InnholdContainer og reduserer manuell styling.

Aksel Page komponent: https://aksel.nav.no/komponenter/primitives/page
Oppgave for å implementere Page komponent: https://favro.com/organization/98c34fb974ce445eac854de0/1844bbac3b6605eacc8f5543?card=NAV-21860

Litt lang forklaring, men håper det gav litt mening 😅